MozillaReality/FirefoxReality/app/src/main/res/values/strings.xml

```diff
@@ -680,6 +755,21 @@
     <!-- This string is displayed in the middle of the Bookmarks List while the bookmarks are being loaded. -->
     <string name="bookmarks_loading">Loading Bookmarks</string>
 
+    <!-- This string is displayed as the title of the folder containing mobile bookmarks (after they're synced). -->
+    <string name="bookmarks_mobile_folder_title">Mobile</string>
+
+    <!-- This string is displayed as the title of the folder containing desktop bookmarks (after they're synced). -->
+    <string name="bookmarks_desktop_folder_title">Desktop Bookmarks</string>
```

Comment:
Wondering if it's expected that the string for mobile above this one appears as just "Mobile" vs this one here where it says "Desktop Bookmarks"?
This actually also applies to the other strings below, where "Bookmarks" doesn't appear either.
I would expect "bookmarks" to be dropped, but really unsure.

Reply:
Yes, that makes sense. I'm going to update to be consistent with Desktop